src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -1,24 +1,36 @@
 package frc.robot.subsystems;
 
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
-public class Intake extends SubsystemBase{
-
-    public Command intake(){
-
-        // Run the motors
+import static frc.robot.Constants.IntakeMotor.*;
+import static frc.robot.Constants.IntakeSpeed.*;
 
-        // Stop the mechanism once the note is in
+public class Intake extends SubsystemBase{
 
-        // Limit switch or beam breaker to check if the note is in so the intake can communicate with the shooter?
+    CANSparkMax intakeMotor;
 
-        // Use pid to run the motors on different speeds?
+    public Intake(){
+        intakeMotor = new CANSparkMax(kIntakeMotorID, MotorType.kBrushless);
+    }
 
-        System.out.print("Return something");
+    @Override
+    public void periodic() {};
 
-        return null;
+    public Command takeIn() {
+        return run(() -> {
+            intakeMotor.set(kIntakeSpeed);
+        });
+    }
 
+    public Command stopMotorCommand() {
```

Comment:
This is good (I remember us going over this bit a couple of meetings ago). Keep this. We'll use it later.

Reply:
👍